### DISCUSSION TITLE
Homepage/Info page needs an update

### DISCUSSION THREAD

**Discussion by rexelbartolome** [freeplane/freeplane#314] [Ideas]
## Homepage/Info page needs an update

With the icons/LaF's update going into stable release soon, I think we should now think about Freeplane's homepage too. I believe it doesn't give justice to what Freeplane looks like right now and it's capabilities.

Right now in SF, the UI of the shown screenshots is very dated (it's for ver 1.5, we're at 1.10) as well as the video. And soon Github will host the downloads/releases too so it might be a good idea to standardize this these two, currently they look like this.

## Freeplane Sourceforge Summary Page

> ![image](https://user-images.githubusercontent.com/88552647/163206262-a9e4c88e-4564-4b01-a177-29541bf0304d.png)

## Github Readme.md

> ![image](https://user-images.githubusercontent.com/88552647/163206342-b63fa307-29a6-4d27-ba04-f355de9684ef.png)

I was thinking more of a **condensed "features" list** would be better instead of a **wall-of-text** changelog or patch notes. Here's an example by Mindmeister

## MindMeister's Feature List

https://user-images.githubusercontent.com/88552647/163209347-cb0f4ca3-7e69-4e59-8d58-c998e4740b87.mp4

This is Xmind's (it's hard to screenshot) : https://www.xmind.net/

## Solution

I don't think we need to follow Xmind's fancy footsteps and have them all animated, but a concise list paired with a gif/static screenshot of the specific feature we're highlighting would be way more appealing than what we have now IMO.

Curious about what our newest addition to the documentation team has to say @ofossum :D but of course everybody's thoughts are welcome, what do you guys think?

**Comment by rexelbartolome** [comment]
Hello all, here's a draft of the highlighted features and/or use cases in no particular order

### A Student's Second Brain
_Attach images from lectures, URL's to websites, and a powerful search and filter system to sift through multiple semester's worth of notes_

### Extensive Formatting Options
_Change fonts, background color, node edges, and node details. Add some personality by inserting icons._

### No More Slides
_Do your brainstorming and presentation in a single program: hide, spotlight, and zoom in on specific nodes during your lecture._

### Powerful Conditional Styling
_Create conditional statements to drive formatting and styling to your nodes._

### Add-ons and scripting aka macros
_Build matrices, compare node contents, export to csv/json/sqlite, import & visualize your browser's bookmarks structure_

### Formulas
_Enter formulas like in a spreadsheet, e.g. reference a value from a different node, sum up child-nodes' values, concatenate several nodes_

### Got A Question?
_Come join us in the discussions in Github!_

### Update: UI Redesign
_Introducing clean and sleek Light Mode and Dark Mode themes as well as modernized icons._

### Update: Command Search
_Search commands using CTRL+F1, execute a command with DOUBLE CLICK or quickly assign a shortcut with CTRL + DOUBLE CLICK._

### User Testimonials
` kinda optional, since we already have a lot to show but in the case we do consider this, we show how other people use FP for different things like CRM, GTD systems, and brainstorming`

### Other Features:

- _Markdown support_ 
- _Node Attributes_
- _Export to HTML, PDF, JPEG, and many others_
- _Mouseless Navigation_
- _Drag 'n Drop Node Reorganization_
- `not sure what other features to add here feel free to share`

---

Now they would each consist of a screenshot of the feature they're highlighting something like this:

![image](https://user-images.githubusercontent.com/88552647/164885816-fbc63f99-f144-4953-a2ee-327020d14341.png)

Now the problem is I don't really know how to show the other features since I don't really use them. So it would be great if you guys can share some screenshots I can use for these features :) 

- Add-ons and scripting aka macros
- Formulas

**Comment by rexelbartolome** [comment]
Here's what I made so far :)

![student](https://user-images.githubusercontent.com/88552647/164890360-5fa94424-cf29-4cb3-ac34-26190d252fa3.png)
![formatting](https://user-images.githubusercontent.com/88552647/164890357-af0be57a-8143-42dd-99d1-348f89dfada5.png)
![styling](https://user-images.githubusercontent.com/88552647/164890361-9a845961-b482-4f78-8f0d-c0ff41332f41.png)
![slides](https://user-images.githubusercontent.com/88552647/164890358-efa121df-cc56-4059-b4d0-9faa7a6c2ce1.png)
![command search](https://user-images.githubusercontent.com/88552647/164890355-129f262d-20c9-43e3-9734-e6aa417a0e3f.png)
![UI](https://user-images.githubusercontent.com/88552647/164890353-3f840af7-2164-40a7-b48e-9a6be98904a4.png)
![discussions](https://user-images.githubusercontent.com/88552647/164890356-e2780b1d-6a64-4f0e-b7cf-beb6855e0c3a.png)



and here's the zip in case the image quality got compressed by github and it's good enough to be used

[freeplane-features.zip](https://github.com/freeplane/freeplane/files/8546682/freeplane-features.zip)

**Comment by ldmpub** [comment]
@rexelbartolome I have added a new Mind Map to Discussion : https://github.com/freeplane/freeplane/discussions/254#discussioncomment-2620322 ; feel free to use it.

**Reply by rexelbartolome** [reply]
thats great! will do :)

**Comment by ofossum** [comment]
# my something-cents

> I believe it doesn't give justice to what Freeplane looks like right now and it's capabilities.
> […]
> Right now in SF, the UI of the shown screenshots is very dated […] And soon Github will host the downloads/releases too so it might be a good idea to standardize this these two, currently they look like this.

Yeeees please! :)
I fully support this and those mock-ups look amazing @rexelbartolome !

I also like the idea of some small animated content next to those features, so that people get a feeling like 
_"Hey, that looks beautiful, is easily accomplished and i can actually use this for my somethingy!"_


# About the other features:

> - Markdown support

I just opened an issue about _exporting mindmaps to Markdown_ (https://github.com/freeplane/freeplane/issues/333) after someone opened an issue about _import Markdown as a mindmap_ (i guess!?; https://github.com/freeplane/freeplane/issues/310)
Therefore: i'd like to see markdown support in a better shape for import / export before highlighting it this way.

> - Node Attributes

Really useful as shown for my bachelors thesis (see https://github.com/freeplane/freeplane/discussions/254#discussioncomment-2555557). 
Makes _filtering_ user-defined stuff much easier!
I don't like the look of it though, but on the other hand: i only used it for this one thing.
So maybe somebody else has a good idea about how to present it.

> - Drag 'n Drop Node Reorganization

This is really a killer-feature for writing notes and connecting things quickly - especially if you're a student.
People i showed Freeplane in action were very impressed by the speed one could make nice notes.

## Filters

I think filters are a very widely used feature, also some don't use them (me as well, but i see why they are great).
I'd like to see them highlighted! 

Maybe something like: 
***Details matter! Focus on what's important and explore connections.***

**Reply by rexelbartolome** [reply]
thanks for the feedback!

i do like it if we can make them animated. though im not sure if gifs or mp4s are supported formats in sourceforge and how big the filesize can be.

ill withdraw the markdown support as a highlight.

i have no idea how attributes work so if you or anyone else could show a screenshot of that too, that would be great :)

yes the filters are very powerful, only got to use them a fair bit recently. though it's a bit hard to show unless animated 🤔
